data_store/controllers/failed_submission.py

```diff
@@ -12,11 +12,11 @@ def get_failed_submission(failure_uuid: str) -> str:
     :return: the failed submission
     """
     try:
-        failure_uuid = UUID(failure_uuid, version=4)
+        uuid = UUID(failure_uuid, version=4)
```

Comment:
nit: don't super love overloading the `uuid` module name. Can we make this function just take a uuid and change it upstream so that we don't need to do the casting/error handling here - or is that more faff?

Reply:
I moved the try - catch outside this function. The function accepts directly a UUID now.

Was this was you suggested?

Reply:
Basically yes :)